modules/training/help/training.md

```diff
@@ -0,0 +1,5 @@
+<h2>Training</h2>
```

Comment:
Headings should use the markdown style '# ' prefix, not HTML, and paragraphs need an empty line between them in markdown.

Reply:
@driusan 
It has been fixed.